.drone.yml

```diff
@@ -34,10 +34,13 @@ steps:
     #- bash ./tests/install_AMP.sh
     - apt-get update && apt-get install -y cmake
     - pip install -r requirements.txt --user -q
+    - pip install -r requirements-extra.txt --user -q
```

Comment:
this is already included in devel below

Reply:
Yep, thanks. 
There is still an issue with drone. It does not give a  green mark even though all tests pass. Somehow sphinx doctest is not sending a success signal.

Reply:
I will try with a newer sphinx version